### DISCUSSION TITLE
https://imagekit.io/docs/api-reference/caching/purge-status

### DISCUSSION THREAD

**Discussion by giscus** [imagekitio/discussions#45] [Docs]
# https://imagekit.io/docs/api-reference/caching/purge-status

This API returns the status of a purge cache request.

https://imagekit.io/docs/api-reference/caching/purge-status

**Comment by tylerlfarkas24-oss** [comment]
https://stage.imagekit.io/docs/api-reference/caching/purge-cache#request-body
